src/namespaces/db/cdc.ts

```diff
@@ -21,20 +21,22 @@ import type { Expr } from "../..";
 import { CypherProcedure } from "../../procedures/CypherProcedure";
 import { normalizeExpr, normalizeList } from "../../utils/normalize-variable";
 
+const CDC_NAMESPACE = "db.cdc";
```

Comment:
I would use the reference so in case the `db` namespace will be renamed then the changes will reflect on the nested namespaces!
```suggestion
const CDC_NAMESPACE = "${DB_NAMESPACE}.cdc";
```

Reply:
This will cause circular dependencies, and the benefit is quite minimal tbh